src/components/AddSighting.js

```diff
@@ -106,7 +106,7 @@ const AddSightingForm = () => {
             />
           </label>
         </div>
-        <input type="submit" value="Add Individual" />
+        <input type="submit" value="Add Sighting" />
```

Comment:
When I can’t decide on wording, I set it as a prop and change it in the component whenever I feel like it!